### DISCUSSION TITLE
account visibility

### DISCUSSION THREAD

**Discussion by b-loo** [community/community#86755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Think account got flagged, is the best course just to make a new one?

**Comment by AlessioBugetti** [comment, marked as answer]
Hi @b-loo , if your account is flagged, contact GitHub Support to find out why and how to fix it. Don't create a new account, it's against the rules and could make things worse.

**Comment by ghostinhershell** [comment]
Hi @b-loo! Echoing @AlessioBugetti's answer above, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them. 

Also, as the user above has mentioned, creating multiple accounts is against our Terms of Service and may result in action against your account:
> One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).

You can check out our Terms of Service here: https://docs.github.com/en/site-policy/github-terms/github-terms-of-service.

**Reply by b-loo** [reply]
Thank you both! I really appreciate it. I realized the fact that you can see this post now and respond means my account has been unflagged.
Cheers!
